cloud/aws/integration.py

```diff
@@ -31,35 +32,26 @@ def vast_import_suricata(c):
     )
 
 
-@task
-def vast_count(c, count):
-    """Validate that the event count in the running server"""
+def vast_count(c):
+    """Run `vast count` and parse the result"""
     res = c.run('./vast-cloud run-lambda -c "vast count"', hide="stdout")
-    vast_count = res.stdout.strip()
-    print(f"Expected vast count {count}, got {vast_count}")
-    assert str(count) == vast_count, "Wrong count"
+    return int(res.stdout.strip())
 
 
 @task
-def clean_context(c):
-    """Remove existing resources that might pertubate the tests"""
-    print("Stop all existing tasks")
-    c.run("./vast-cloud stop-all-tasks")
-    print("Clean mounted folder")
-    c.run('./vast-cloud run-vast-task --cmd-override "rm -rf /var/lib/vast/*"')
-    print("Waiting for the cleaning task to boot...")
-    time.sleep(100)
+def vast_data_import(c):
+    """Import data from a file and check count increase"""
+    print("Import data into VAST")
+    init_count = vast_count(c)
+    vast_import_suricata(c)
+    new_count = vast_count(c)
+    print(f"Expected count increase 7, got {new_count-init_count}")
+    assert 7 == new_count - init_count, "Wrong count"
```

Comment:
I did it this way because I liked the fact that it also displayed when there wasn't an error. It gives a visual feedback that your tests are running successfully (this is usually provided by the test framework, but we don't use one here)